_posts/en/newsletters/2021-05-26-newsletter.md

```diff
@@ -0,0 +1,220 @@
+each week's newsletter to try to ensure the content we produce is accurate and
+clear.
+
+Special thanks to [Shigeyuki Azuchi][], who translates our newsletters into
+Japanese, and [Akio Nakamura][] who has also translated and reviewed our
+Japanese material.
+
+Thanks to all the members of the Bitcoin community -- too numerous to name
+individually -- who have reviewed our newsletters, helped us understand
+concepts, and opened issues and PRs when we've made mistakes.
+
+All of this work is made possible by our generous [supporters][], primarily
+our [founding sponsors][] - Wences Casares, John Pfeffer and Alex Morcos.
+
+Finally, thank you, our readers. We love being part of this community and
+contributing to this ecosystem. Knowing how valuable this resource is to so many
+people, and hearing feedback from our readers is hugely rewarding for us. If you
+want to contribute, or have suggestions for how we can improve, please don't
+hesitate to contact us at [info@bitcoinops.org][info].
+
+## Selected Q&A from Bitcoin StackExchange
+
+*[Bitcoin StackExchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Eclair 0.6.0][] FIXME:harding
+
+- [LND 0.13.0-beta.rc2][LND 0.13.0-beta] is a release candidate that
+  adds support for using a pruned Bitcoin full node, allows receiving
+  and sending payments using Atomic MultiPath ([AMP][topic multipath payments]),
+  and increases its [PSBT][topic psbt] capabilities, among other improvements
+  and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+
+- [Bitcoin Core #21843][] rpc: enable GetAddr, GetAddresses, and getnodeaddresses by network FIXME:jnewbery
+
+- [Eclair #1810][] Make payment_secret mandatory FIXME:dongcarl
+
+- [Eclair #1774][] extends Java's built-in `SecureRandom()` [CSPRNG][]
+  function with a secondary source of weaker randomness.  The weaker
+  randomness is hashed and the hash digest xored with the primary
```

Comment:
(maybe) s/xored/XORed/

Reply:
I don't think we've historically uppercased other boolean terms, so we probably shouldn't start now.  If you're really worried about this, I can rephrase to use "xor" without a suffix.

Reply:
(FWIW, I just searched Google and DuckDuckGo for "xored" and got reasonable results, so anyone who does get confused should be able to straighten themselves out pretty quickly.)

Reply:
not worried about it, so long as no one thinks we meant to write "bored" instead :stuck_out_tongue: